pkgs/by-name/ly/LycheeSlicer/package.nix

```
@@ -39,16 +41,21 @@ appimageTools.wrapType2 {
     install -Dm444 -t $out/share/applications ${desktopItem}/share/applications/*
   '';
 
-  extraLibraries = [
+  extraPkgs = pkgs: with pkgs; [
```

```suggestion
  extraPkgs = [
```
you don't have `pkgs` inside the recipe, have you tried building it? this line must've errored